packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -56,91 +57,124 @@ export interface DrugOrderFormProps {
   promptBeforeClosing: (testFcn: () => boolean) => void;
 }
 
-const comboSchema = {
-  value: z.string(),
-  valueCoded: z.string(),
-  default: z.boolean().optional(),
-};
+const createMedicationOrderFormSchema = (requireOutpatientQuantity: boolean) => {
```

Comment:
Wouldn't it be simpler to pass in the `t()` function here rather than using `translateFrom()` and all the comments? It was slightly different with the previous structure, I think.

Reply:
Yes, it would!